sys/include/net/gnrc/ipv6.h

```diff
@@ -181,6 +181,50 @@ extern "C" {
 #define CONFIG_GNRC_IPV6_STATIC_LLADDR_IS_FIXED 0
 #endif
 
+/**
+ * @brief   Select interfaces by driver types for setting static link local
+ *          addresses
+ *
+ * This option allows to explicitly include interfaces by matching their
+ * netdev driver types, encoded in a bitmask.
+ * See @ref netdev_type_t for possible values of netdev driver types.
+ * Matching NETDEV_ANY will include all netdev driver types.
+ *
+ * Example usage, includes AT86RF215 and MRF24J40 driver types:
+ *
+ * @code{.c}
+ * #define CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK \
+ *    ((1UL << NETDEV_AT86RF215) | (1UL << NETDEV_MRF24J40))
+ * @endcode
+ *
+ * A value of 0 switches the feature off.
+ */
+#ifndef CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK
+#define CONFIG_GNRC_IPV6_STATIC_LLADDR_NETDEV_MASK 0UL
```

Comment:
I would have set it `(~NETDEV_ANY)`, so any `(1u << (netdev_type_t))` would be set.

Reply:
It may seem obvious but you get the exclude semantic  by writing `(~NETDEV_ANY) & ~(1u << NETDEV_AT86RF215)` for example.

Reply:
It is not obvious without looking it up that `NETDEV_ANY` is 0. Therefore and because availability of the netdev driver type id in the netdev struct depends on `MODULE_NETDEV_REGISTER`, I would prefer to explicitly switch this feature on. Otherwise `MODULE_NETDEV_REGISTER` must always be included.